model/patch/patch_test.go

```diff
@@ -750,3 +751,64 @@ func checkEqualVTs(t *testing.T, expected []VariantTasks, actual []VariantTasks)
 		assert.True(t, found, "build variant '%s' not found", expectedVT.Variant)
 	}
 }
+
+func TestSetParametersFromParent(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(Collection))
+	parentPatchID := mgobson.NewObjectId()
+	parentPatch := Patch{
+		Id: parentPatchID,
+		Triggers: TriggerInfo{
+			DownstreamParameters: []Parameter{
+				{
+					Key:   "hello",
+					Value: "notHello",
+				},
+			},
+		},
+	}
+	assert.NoError(parentPatch.Insert())
+	p := Patch{
+		Id: bson.NewObjectId(),
+		Triggers: TriggerInfo{
+			ParentPatch: parentPatchID.Hex(),
+		},
+	}
+	assert.NoError(p.Insert())
+	assert.NoError(p.SetParametersFromParent())
+	assert.Equal(parentPatch.Triggers.DownstreamParameters[0].Key, p.Parameters[0].Key)
+}
+
+func TestSetDownstreamParameters(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(Collection))
+
+	p := Patch{
+		Id: mgobson.NewObjectId(),
```

Comment:
could this use `bson.NewObjectId()` too?

Reply:
I'm assuming that it would be the same thing because the import is `mgobson "gopkg.in/mgo.v2/bson" `